requirements_test.txt

```diff
@@ -0,0 +1,4 @@
+mock-serial==0.0.1
+pytest==8.4.1
+pytest-asyncio==1.1.0
+pytest-reserial==0.4.3
```

Comment:
Delete this file, dependencies are defined in `pyproject.toml`:

```
-dev = [{include-group = "typing"}, {include-group = "linting"}, "pre-commit"]
+tests = ["mock-serial", "pytest", "pytest-asyncio", "pytest-reserial"]
+dev = [{include-group = "typing"}, {include-group = "linting"}, {include-group = "tests"}, "pre-commit"]
```

Then just run `uv sync`